src/UnityExtension/Assets/Editor/GitHub.Unity/UI/GitPathView.cs

```diff
@@ -0,0 +1,286 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEditor;
+using UnityEngine;
+
+namespace GitHub.Unity
+{
+    [Serializable]
+    class GitPathView : Subview
+    {
+        private const string GitInstallTitle = "Git installation";
+        private const string PathToGit = "Path to Git";
+        private const string GitPathSaveButton = "Save Path";
+        private const string GitInstallFindButton = "Find install";
+        private const string BrowseButton = "...";
+        private const string GitInstallBrowseTitle = "Select git binary";
+        private const string ErrorInvalidPathMessage = "Invalid Path.";
+        private const string ErrorGettingSoftwareVersionMessage = "Error getting software versions.";
+        private const string ErrorMinimumGitVersionMessageFormat = "Git version {0} found. Git version {1} is required.";
+        private const string ErrorMinimumGitLfsVersionMessageFormat = "Git LFS version {0} found. Git LFS version {1} is required.";
+
+        [SerializeField] private string gitExec;
+        [SerializeField] private string gitExecParent;
+        [SerializeField] private string gitExecExtension;
+        [SerializeField] private string newGitExec;
+        [SerializeField] private bool isValueChanged;
+        [SerializeField] private bool isValueChangedAndFileExists;
+        [SerializeField] private string gitFileErrorMessage;
+        [SerializeField] private string gitVersionErrorMessage;
+
+        [NonSerialized] private bool isBusy;
+        [NonSerialized] private bool gitExecHasChanged;
+
+        public override void OnEnable()
+        {
+            base.OnEnable();
+
+            gitExecHasChanged = true;
+        }
+
+        public override void OnDataUpdate()
+        {
+            base.OnDataUpdate();
+            MaybeUpdateData();
+        }
+
+        public override bool IsBusy
+        {
+            get { return isBusy; }
+        }
+
+        public override void OnGUI()
+        {
+            // Install path
+            GUILayout.Label(GitInstallTitle, EditorStyles.boldLabel);
+
+            EditorGUI.BeginDisabledGroup(IsBusy || Parent.IsBusy);
```

Comment:
If we always have to check the value of the parent, we should either be setting it directly or the `IsBusy` property should be returning the aggregate status so that `IsBusy` always reflects the state of the current view and the hierarchy up the chain. Otherwise it's an opportunity to forget to do it in some place.

Reply:
I had the same feelings, but I couldn't figure out a way to do it that I was truly happy with.
Right now `IsBusy` represents self and children.